bc_obps/compliance/models/compliance_summary.py

```diff
@@ -70,6 +70,37 @@ def compliance_status(self) -> str:
         else:
             return self.ComplianceStatus.OBLIGATION_FULLY_MET
 
+    @property
+    def obligation_id(self) -> str:
```

Comment:
This logic should be in a service of some sort, and an `obligation_id = CharField(...)` on the model